### PR TITLE
Releases: blog post & release notes for 0.15.0

### PR DESCRIPTION
**Pull request by harding**
Preview: http://dg0.dtrt.org/en/2017/09/01/release-0.15.0/

Todo:

- [x] Set date on blog post to actual release date (change filename and edit YAML metadata)
- [x] Add sha256sums to bottom of blog post

I'd especially appreciate review of [this part](http://dg0.dtrt.org/en/2017/09/01/release-0.15.0/#the-future-p2sh-wrapped-segwit-addresses) about the plans for upcoming default segwit address support in the wallet.

### CODE REVIEW

File: _posts/en/posts/2017-09-01-release-0.15.0.md [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+
+- **30% to 40% faster block validation and 10% to 20% less memory used** on tests of Initial Block Download (IBD), with far fewer writes to disk.  This is the result of simplifying the format of the the chainstate database that tracks each spendable group of bitcoins and what information the owner of those bitcoins needs to provide in order to spend them.
+
+- **40% to 50% faster validation of blocks consisting of previously-seen transactions** as the result of repeating fewer validation steps when a previously-verified mempool transaction is later received in a block.
+
+- **Moderate performance gains on some platforms** as the result of using hardware acceleration for some operations, such as support on modern computer processors for single-SHA256 hashing which results in about a 50% improvement each time one of Bitcoin's double-SHA256 hashes needs to be computed.  This mainly benefits users of 64-bit Intel and AMD processors produced in 2008 or later.
```
Comment (TheBlueMatt) [inline _posts/en/posts/2017-09-01-release-0.15.0.md]: Hmm, this is misleading due to the fact that this is disabled-by-default (and in the release binaries). Maybe mention it is just "in testing" or so?

File: _posts/en/posts/2017-09-01-release-0.15.0.md [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+
+Bitcoin Core has supported creating segwit addresses since 0.13.0, but this support was designed for testing has only been available to expert users---we were waiting to see if segwit was adopted before adding segwit support to the regular user interfaces, both graphical and RPC.
+
+The timing of segwit lock in and activation meant that we had to choose between either delaying the planned release of 0.15.0 and all its features described above or shipping 0.15.0 without a user interface defaulting to segwit.
+
+We decided to take the later option, but we're also not going to wait the normal six months before the next major update.  Instead, our next release will be a feature release that generates segwit-compatible addresses by default.  This will be made available as soon as it has been written and thoroughly tested.
+
+For those of you interested in technical details, our plan is to use P2SH-wrapped segwit addresses that are compatible with nearly all other wallets on the network.  Bech32 native segwit addresses will either be implemented in a subsequent release (assuming no problems arise) or will only be available through expert options for now.
```
Comment (TheBlueMatt) [inline _posts/en/posts/2017-09-01-release-0.15.0.md]: I believe the plan is to support Bech32 native segwit addresses for sending only until the rest of the community updates.

File: _posts/en/posts/2017-09-01-release-0.15.0.md [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+
+## The future: P2SH-wrapped segwit addresses
+
+As final preparations are being made to release Bitcoin Core 0.15.0, segregated witness has activated on the Bitcoin network and is now ready to use.
+
+Bitcoin Core has supported creating segwit addresses since 0.13.0, but this support was designed for testing has only been available to expert users---we were waiting to see if segwit was adopted before adding segwit support to the regular user interfaces, both graphical and RPC.
+
+The timing of segwit lock in and activation meant that we had to choose between either delaying the planned release of 0.15.0 and all its features described above or shipping 0.15.0 without a user interface defaulting to segwit.
+
+We decided to take the later option, but we're also not going to wait the normal six months before the next major update.  Instead, our next release will be a feature release that generates segwit-compatible addresses by default.  This will be made available as soon as it has been written and thoroughly tested.
```
Comment (TheBlueMatt) [inline _posts/en/posts/2017-09-01-release-0.15.0.md]: Maybe "our next feature release will generate..." instead? It is possible (though hopefully unlikely) that we have to do a bugfix release between now and then.

File: _posts/en/posts/2017-09-01-release-0.15.0.md (hunk 1 above)
Comment (harding) [reply _posts/en/posts/2017-09-01-release-0.15.0.md]: I don't feel comfortable saying something is in testing in a post about a *release*, so in the edit commit I'm about to push, I changed the example to mention the LevelDB CRC acceleration instead.  It's less of a cool example IMO, but thanks for not letting me get away with misleading readers.

**Comment by laanwj** [comment]
ACK, thanks for writing this

**Comment by TheBlueMatt** [comment]
Looks like this just needs the edate to be set?

**Comment by harding** [comment]
Date updated and hashes from https://bitcoincore.org/bin/bitcoin-core-0.15.0/SHA256SUMS.asc added.

**Comment by harding** [comment]
Oh, forgot about the release notes.  Those have now been updated as well.

**Comment by gmaxwell** [comment]
ACK.

**Pull request merged**